# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/tests/reftest/reftest.list
# Commit: 66e788b91a78
# Full Hash: 66e788b91a785ecda515f8a25dc56acab6fc2d63
# Author: Cristina Horotan <chorotan@mozilla.com>
# Date: 2025-05-08 09:24:40
# Regressor Bug: 1961387
# File Overlap Count: 7
# Description:
#   Revert "Bug 1961387 - Add a reftest with larger glyphs, for CBDT version of Noto Color Emoji. r=gfx-reviewers,lsalzman" for causing crashes (bug 1965123)
#   
#   This reverts commit 05d7f427e840f62d7cd9eaffe83df511aad7aa08.
#   
#   Revert "Bug 1961387 - Account for scaling of color-bitmap fonts when setting up the font-unit scale factor. r=gfx-reviewers,lsalzman"
# ==============================================================================

diff -r 82516541b4b1 -r 66e788b91a78 gfx/tests/reftest/reftest.list
--- a/gfx/tests/reftest/reftest.list	Thu May 08 09:06:49 2025 +0300
+++ b/gfx/tests/reftest/reftest.list	Thu May 08 09:13:55 2025 +0300
@@ -48,6 +48,4 @@
 != 1870240-colrv1-cycle.html 1870240-colrv1-cycle-notref.html
 == 1909933.html 1909933-ref.html
 fuzzy(0-2,0-6229) random-if(Android) == 1912431-emoji-globalAlpha.html 1912431-emoji-globalAlpha-ref.html
-== 1961387-1.html 1961387-1-ref.html
-== 1961387-2.html 1961387-2-ref.html
 skip-if(!cocoaWidget) == 1964225-1.html 1964225-1-ref.html